src/core/obj.js

```diff
@@ -254,6 +254,69 @@ class Catalog {
     return permissions;
   }
 
+  get optionalContentConfig() {
+    let config = null;
+    try {
+      const properties = this.catDict.get("OCProperties");
+      if (!properties) {
+        return shadow(this, "optionalContentConfig", null);
+      }
+      const groupsData = properties.get("OCGs");
+      if (!Array.isArray(groupsData)) {
+        return shadow(this, "optionalContentConfig", null);
+      }
+      const groups = [];
+      const groupRefs = [];
+      // Ensure all the optional content groups are valid.
+      for (const groupRef of groupsData) {
+        if (!isRef(groupRef)) {
+          continue;
+        }
+        groupRefs.push(groupRef);
+        const group = properties.xref.fetchIfRef(groupRef);
+        groups.push({
+          id: groupRef.toString(),
+          name: group.get("Name"),
```

Comment:
Since this is, I believe, a string; don't you need to wrap it in `stringToPDFString()` to avoid issues?